src/React/DOM/Props.purs

```diff
@@ -297,6 +300,19 @@ radioGroup = unsafeMkProps "radioGroup"
 readOnly :: Boolean -> Props
 readOnly = unsafeMkProps "readOnly"
 
+ref :: String -> Props
+ref = unsafeMkProps "ref"
+
+-- | You can use `writeRef` to store a reference on `Refs`.
+-- | ``` purescrript
+-- | div [ refCb (writeRef this "inputEl") ] [...]
+-- | ```
+refCb
+  :: forall access eff
+   . (Ref -> Eff (refs :: ReactRefs (write :: Write | access) | eff) Unit)
```

Comment:
Just as a note, I think this may need to be

```purescript
(Nullable Ref -> Eff (refs :: ReactRefs (write :: Write | access) | eff) Unit)
```

Reply:
From react docs:

React will call the ref callback with the DOM element when the component mounts, and call it with null when it unmounts.

And this is done to avoid memory leaks (DOM references used to leak in old IEs).  If we write the ref only if it's not `null`, you'll prevent the original mechanism to prevent memory leaks.

So if we change this type signature (which I do like, since things are explicitly typed then) then we also need to change `writeRef :: forall p s. ReactThis p s -> String -> Nullable Ref -> Eff ...`

Reply:
Agreed. I think it makes sense to write a `Nullable Ref`.